src/editor/EditorControls.scss

```diff
@@ -1,6 +1,5 @@
 .editor-controls {
   padding: 5px;
```

Comment:
Thought: We should probably have some sort of standardized unit size and set that as a variable, then scale that unit accordingly. Right now the mockups are based on 4px spacing, so that might be a good place to start.

Reply:
I *believe* Bootstrap has some kind of default spacing variable. We can set that to 4px, and then go from there.

Reply:
Well turns out, $spacer = 1rem = 16px, and the first spacing increment is 0.25 * $spacer = 4px. Sometimes life does work out that way.

Reply:
Found it: https://getbootstrap.com/docs/4.4/getting-started/theming/#sass-options

Looks like the default spacing variable (`$spacer`) is meant for margins and so is set to a relatively large value (1rem). We may just want to set it to 16px so that the minimum spacing unit is 4px; alternatively, we can set `$spacer` to 4px, and change the size mappings in the `$spacers` map (https://getbootstrap.com/docs/4.4/utilities/spacing/#notation).

Reply:
Well, `mb-1` is basically 4px, since its `$spacer * 0.25`. Think we can leave the mapping the way it is for now.